ToolsTest/Володимир/nUnit.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using Tools;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NUnit.Framework;
+using Xunit;
+using Assert = Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
+using nAssert = NUnit.Framework.Assert;
+using xAssert = Xunit.Assert;
+using Theory = Xunit.TheoryAttribute;
```

Comment:
As you do not use different frameworks in the this file, remove the type aliases.
The same in the other test files.
In this case in fact you are using the nUnit test framework, but for checking the results you are using the MsTest Assert class. So, you are using the both test frameworks in each test.